apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/service/appdata/impl/TwoFactorAuthServiceImpl.java

```diff
@@ -48,7 +48,9 @@ public TwoFactorAuthServiceImpl(@Named("DBRepository") TwoFactorAuthRepository r
     public TwoFactorAuthServiceImpl(TwoFactorAuthRepository repository, String issuerSuffix, Random random,
                                     TwoFactorAuthRepository targetFileRepository) {
         if (issuerSuffix == null || issuerSuffix.trim().isEmpty()) {
-            issuerSuffix = RuntimeEnvironment.getInstance().isDesktopApplicationEnabled() ? "desktop" : "web";
+          //TODO: what is the difference?  
+          //  issuerSuffix = RuntimeEnvironment.getInstance().isDesktopApplicationEnabled() ? "desktop" : "web";
```

Comment:
Issuer suffix required for 2fa qr codes, which will create two separate entries in Google Authenticator for same account and will not replace the existing, when 2fa was enabled on desktop, but was already enabled on web before